prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -27,17 +27,6 @@ class BlobAccess(
     private val hl7Serializer: Hl7Serializer,
     private val redoxSerializer: RedoxSerializer
 ) : Logging {
-    private val defaultConnEnvVar = "AzureWebJobsStorage"
```

Comment:
It looks like there's allot going on here but it's only moving anything that didn't need an instance of `BlobAccess` to the companion singleton.